pkg/cmd/clientConfig.go

```diff
@@ -84,8 +87,13 @@ kubectl plugin mobile get clientconfig`,
 				var err error
 				includedService := true
 				for _, excluded := range client.Spec.ExcludedServices {
-					if strings.TrimSpace(excluded) == strings.TrimSpace(svc.ID) {
+					catalogService, err := ccc.scClient.ServicecatalogV1beta1().ServiceInstances(ns).Get(excluded, v1.GetOptions{})
+					if err != nil {
```

Comment:
Not sure we should continue here as if there is no service instance, there should be no service and no secret and no configmap. If there is an error getting serviceinstances for some reason and we ignore it we could accidentally give the config to a client that shouldn't have it

Reply:
in that case we should check if the err is a NotFoundErr and continue otherwise fail. I think you can do that via kerror "k8s.io/apimachinery/pkg/api/errors"

Reply:
ok, updated that now.